Release notes 0.115.0 (#14522)
home-assistant/home-assistant.github.io: source/_posts/2020-09-17-release-115.markdown

change the lights and pick some music.

Any media player can enable this new feature. This release adds media browser
support to [Arcam FMJ](/integrations/arcam_fmj), [Kodi](/integrations/kodi),
[Philips JS](/integrations/philips_js), [Plex](/integrations/plex),
[Roku](/integrations/roku), [Sonos](/integrations/sonos)
and [Spotify](/integrations/spotify)!

Thanks to [@jjlawren] for the initial implementation and to [@cgtobi],
[@ctalkington], [@martinhjelmare], [@elupus] for the work on the various
integrations and backend refining.

But a media browser isn't just backend, it also needs to look good. Thankfully,
we had [@NKDZCK] on the case, with the help of [@bramkragten]. They came up with
a beautiful design so you can browse your media in style.

[@nkdzck]: https://github.com/NKDZCK

<div class="videoWrapper">
  <iframe width="853" height="480" src="https://www.youtube-nocookie.com/embed/4tQvMezHJkQ" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
</div>

When [@hunterjm] saw the new media browser, he got an idea. What if the user can
use the media browser to browse media offered by any integration and play it on
any of their media players? And so the

> [the media source integration](/integrations/media_source/) was born.

Reviewer: ```suggestion
any of their media players? And so
[the media source integration](/integrations/media_source/) was born.
```

updated. In this release, we've updated [the Chromecast integration](/integrations/cast/).
If you click its media browser button, you will be presented with the available
local sources.

Sometimes you just want to play your media without having to turn on the TV. To
cover that case, [@NKDZCK] added a new media browser to the Home Assistant
interface, which can be used to play your media. That way, you can quickly see
what that motion event that you just received was about.

<div class="videoWrapper">
  <iframe width="853" height="480" src="https://www.youtube-nocookie.com/embed/yvbjeLwXo30" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
</div>

## Home Assistant Tags

<p class='img'>
<img src='/images/blog/2020-09-15-home-assistant-tags/tag-reader.jpg' alt='Photo of the Home Assistant tag reader made by Adonno'>
The Home Assistant tag reader, made by Adonno with the help of MagnusO.
</p>

Home Assistant has now native support for tags! With Home Assistant Tags, we’re
making scannable tags (NFC/RFID) a first-class citizen in Home Assistant.
Easy to read, write and automate!

We have a beautiful UI for it in the frontend, our mobile companion apps have

> been updated to work seamlessly with tags and there is now even a open source

Reviewer: ```suggestion
been updated to work seamlessly with tags and there is now even an open source
```

scanner available, as shown in the image above.

For all details, [read the dedicated blog post about Home Assistant tags][tags]!

[tags]: /blog/2020/09/15/home-assistant-tags/

## Customize the sidebar

One of the most requested functions of What the heck was customizing the

> sidebar, you do want history for your entities in the more info dialog, but you

Reviewer: ```suggestion
sidebar: You do want history for your entities in the more info dialog, but you
```

</p>

You can enter edit mode by pressing and holding the sidebar's header or from
your profile page. You can then drag the items in the order you want them
or remove them by clicking the `x` next to it.

The removed items will be visible at the bottom of the list, by clicking the
`+` button, you can add them back to the sidebar.

These settings are stored on your device in the local storage of your browser,
this means that this setting is device-based and will be cleared when you logout.

## Person image upload

You can now upload images for a person in the frontend!

Select or drop an image in the input field and then crop it to a square. The
image is stored on your Home Assistant server.

<div class='videoWrapper'>
<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/rOlRnwaaT7Y" frameborder="0" allowfullscreen></iframe>
</div>

We will use the image in the frontend for your persons and the sidebar for the
user linked to this person.

This feature is powered by the new image integration and opens up for future
possibilities. For example, uploading images, for use in your Lovelace picture
cards, would be nice!

## Updated more info dialog

> The more info dialog is updated. In case an entity has controls, for example,

Reviewer: ```suggestion
The more info dialog is updated. In case an entity has controls, for example
```

let's say you left your garage door open and you leave the "Home" zone. Home
Assistant could send you an actionable notification, saying: "Hey, you left
the garage door open, shall I close if for you?" with 2 choices: Yes/No.

Where you previously would have needed 3 automations for this. The first
for sending a notification and two others for handling the "Yes" or "No"
answer. Using the `wait_for_trigger` this could be done in a single automation.

Right after sending the notification, the `wait_for_trigger` could halt the
script from continuing, until it receives the "Yes" or "No" answer and continue
after that and run the actions you'd like based on the answer received.

This new feature is not just for our YAML uses; it also is added to our
automation editor in the UI.

<p class='img'>
<img src='/images/blog/2020-09-0.115/wait-for-trigger.png' alt='Screenshot of the new wait_for_trigger in the automation editor'>
The automation editor can also use the new Wait for trigger action.
</p>

### Triggers & Conditions on entity attributes

Every tried to create an automation trigger or condition on an entity attribute?

> Like the temperature of a climate or weather entity?

Reviewer: ```suggestion
Ever tried to create an automation trigger or condition on an entity attribute, like the temperature of a climate or weather entity?
```

that forum this month!

And it is not just that, a lot of people jumped in fixing these things as well!
What a community! The month of WTH is almost over now, after that we will
close the category on the forum, until the next WTH month.

Besides that, two totally new and big features are added this release, which
we really wanted to polish before releasing it.

Normally I write some things about the release in my personal introduction
note right here, but honestly, I have no idea where to start... it is just
too darn much! I personally love the new automation features, but there are so
many of those in this release as well. 😅 I'm not going to try writing this. 😂

So, let me close with a thank you to our founding father:

> Paulus, thank you so much for what you have imagined, started and created 7
> years ago. Your idea has changed the lives of many, including my own.
> Thank you.

Of course, it is not just Paulus, but everybody else who contributes to the
project in any way. Code, text, support, chat, YouTube video, live streams,
blogs, articles, community guides, documentation, tweets, issues, bug reports,

> feature requests, ideas, questions or even if you just using it.

Reviewer: ```suggestion
feature requests, ideas, questions or even if you just use it.
```

<br />
It's a celebration! 🎉
</blockquote>

~ [_Kool & The Gang - Celebration_](https://www.youtube.com/watch?v=3GwjfUFyY6M)

7! Siete! Soch! Syv! Sieben! Seitsemän! Cедем! Sept! Sju! επτά! Zeven! Sette!
Seven! Years old today! And ooooh, are we going to party and celebrate with
you!

You might have noticed, this release is a bit late, two weeks late to be exact.
Our regular release cycle is 3 weeks, this time however, we took 5 weeks.
Sure, the reason was of course related to our birthday today, but what is a
better gift than a jam packed release that has something for everybody?

Thankfully, a lot of you have been sharing their gift wishlist items during
[the month of "What the heck?!"][wth] (WTH). I love you all! What a good and
amazing set of great ideas, annoyances and other suggestions have been made in
that forum this month!

And it is not just that, a lot of people jumped in fixing these things as well!
What a community! The month of WTH is almost over now, after that we will
close the category on the forum, until the next WTH month.

Besides that, two totally new and big features are added this release, which

> we really wanted to polish before releasing it. The additional time this release
> was very well spent.

Reviewer 2: ```suggestion
we really wanted to polish before releasing it. The additional time needed for 
this release was very well spent.
```

Author: Removed that part.

Normally I write some things about the release in my personal introduction
note right here, but honestly, I have no idea where to start... it is just
too darn much! I personally love the new automation features, but there are so
many of those in this release as well. 😅 I'm not going to try writing this. 😂

So, let me close with a thank you to our founding father:

> Paulus, thank you so much for what you have imagined, started and created 7
> years ago. Your idea has changed the lives of many, including my own.
> Thank you.

Of course, it is not just Paulus, but everybody else who contributes to the
project in any way. Code, text, support, chat, YouTube video, live streams,
blogs, articles, community guides, documentation, tweets, issues, bug reports,
feature requests, ideas, questions or even if you just using it.
Thank you for contributing! ❤️

Happy birthday and enjoy the release!

../Frenck

[wth]: /blog/2020/08/18/the-month-of-what-the-heck/

## Table of contents

Alright, this release is massive, so here is a table of contents for helping

> navigating your around this release.

Reviewer 2: ```suggestion
navigating you around this release.
```

</p>

### Triggers & Conditions on entity attributes

Every tried to create an automation trigger or condition on an entity attribute?
Like the temperature of a climate or weather entity?

You used to need to use a template for that. Either by extracting the attribute
from an entity using a template sensor or by writing a template condition.
A reason for a lot of you to put it up as a "What the heck?!" entry. And you
know what? You guys are right, this was difficult.

<p class='img'>
<img src='/images/blog/2020-09-0.115/attributes.png' alt='Screenshot of using attributes in the automation editor'></a>
The automation editor now supports attributes on triggers and conditions.
</p>

So, Home Assistant now support the use of attributes in triggers & conditions.
Both the state and numeric state, triggers & conditions, now have a `attribute`
option that can be set with the attribute to use. The UI got a nice field for
it, as shown in the above screenshots, but of course, it is also available
when you are using YAML for your automations.

Some example triggers and conditions using attributes in YAML:

```yaml
trigger:
  - platform: state
    entity_id: climate.living_room

>     attribute: havc_action

Reviewer 2: `    attribute: hvac_action`

>     attribute: havc_action

Reviewer 2: ```suggestion
    attribute: hvac_action
```

    below: 80
```

### Use `input_datetime` helpers in automation triggers

Using dates and times in your automation can be hard. If often needs quite a bit
of Jinja templating and is actually really hard to do. [@pnbruckner] noticed,

> and he added the possibility use your `input_datetime` helper entities

Reviewer: ```suggestion
and he added the possibility to use your `input_datetime` helper entities
```

### Use `input_datetime` helpers in automation triggers

Using dates and times in your automation can be hard. If often needs quite a bit
of Jinja templating and is actually really hard to do. [@pnbruckner] noticed,
and he added the possibility use your `input_datetime` helper entities
directly in time trigger!

Assume you have an `input_datetime.bedroom_alarm_clock_time` helper entity, that
is in your Lovelace UI, which you can set a time in. Great! You can now
just use it in your automations to trigger on:

```yaml
trigger:
  - platform: time
    at: input_datetime.bedroom_alarm_clock_time
```

Yes, he made it that elegant. It also works for multiple, or mixed variable and
statically set times.

```yaml
trigger:
  - platform: time
    at:
      - "10:00"
      - input_datetime.bedroom_alarm_clock_time
      - input_datetime.some_other_time_entity
```

More about the time trigger, can be found in our [documentation][trigger-time].

[trigger-time]: /docs/automation/trigger/#time-trigger

### Use `input_*` helpers in conditions

[@pnbruckner] set a standard in the above, and we used that to start working
on making this something that would work on more places. As of this release,

> all `input_*` can be used in conditions.

Reviewer: ```suggestion
all `input_*` entities can be used in conditions.
```

The `time` condition can accept `input_datetime` helper entities, similar to
the trigger shown above.

```yaml
conditions:
  - condition: time
    after: input_datetime.house_silent_hours_start
    before: input_datetime.house_silent_hours_end
```

The `numeric_state` condition now accepts `input_number` helper entities for
the `above` and `below` options.

```yaml
conditions:
  - condition: numeric_state
    entity_id: climate.living_room_thermostat
    attribute: temperature
    above: input_number.temperature_threshold_low
    below: input_number.temperature_threshold_high
```

> And finally, the `state` condition accepts any `input_*` helper entit in its

Reviewer: ```suggestion
And finally, the `state` condition accepts any `input_*` helper entity in its
```

```

{% endraw %}

For a more extensive example, check out the example written in [the blog
article about Home Assistant Tags][tags].

### Other scripts and automation changes

But wait! There is more! 😂

There was no way of knowing if a wait template was timed out or if it continued
normally. Now, we do know this. After each wait template, a new variable is
available: `wait`. It provides `wait.completed` (indicates if the template
evaluated to true before the timeout expired) and `wait.remaning` (remaining time
out).

{% raw %}

```yaml
sequence:
  - wait_template: "{{ is_state('binary_sensor.abc', 'on') }}"
    timeout: 10
    continue_on_timeout: true
  - choose:
      - conditions:
          - condition: template
            value_template: "{{ not wait.completed }}"
        sequence:
          # Handle timeout case
```

{% endraw %}

The new script and automation run modes are amazing! But in some cases, they
might be polluting your logs. For example, you have a automation in `single`
mode, but it does get triggered multiples times sometimes and you are not
interested in the log. You can now control that with the `max_exceeded` option.

> The example below silents the automation and will not log when it gets

Reviewer: ```suggestion
The example below silences the automation and it will not log when it gets
```

triggered while it was already running:

```yaml
automation:
  - trigger: ...
    max_exceeded: silent
    action: ...
```

## Calendar card

Like promised, when we introduced the calendar panel, we now also added a
Lovelace calendar card!

This allows you to create as many calendars as you want with the entities you
want.

If you want multiple calendar panels, create a Lovelace dashboard with a
panelmode view with a calendar card!

<p class='img'>
<img src='/images/blog/2020-09-0.115/calendar-card.png' alt='Screenshot of the new calendar card'>
Screenshot of the new calendar card.
</p>

## Template developer tools

The template developer tools are very useful for checking if the template you
made works and does what you want it to do. But people had some annoyances with
the tool; the editor is always filled with sample data, that can give a lot of
response. It would be more useful to have your previously used template there.
Yes, you guessed it, another WTH!

The template would also not automatically re-render after the state of an entity

> is changed. Causing you needed to change the template in order to re-render it.

Reviewer: ```suggestion
is changed, causing you needing to change the template in order to re-render it.
```

> We addressed both these issues, we save your last-used template and will show

Reviewer: ```suggestion
We addressed both these issues. We save your last-used template and will show
```

that instead of the example when you visit the template developer tools.

<p class='img'>
<img src='/images/blog/2020-09-0.115/dev-tools-templates.gif' alt='Screenrecording the live state changes in the template development tools.'>
Templates you write in the developer tools, are now updating live!
</p>

We will also listen for changes of the entities you used in your template and
automatically re-render your template. As a bonus, we will show
which entities Home Assistant detected you are using in your template.

## Reload everything YAML

WTH, do we still need to restart Home Assistant for applying YAML configuration?
That was one of the WTH raised. It is being worked on!

> This release, [@bdraco] found a way to reload some of internal integrations

Reviewer: ```suggestion
This release, [@bdraco] found a way to reload some of the internal integrations
```

and boosted this capability to a lot of integrations. For those, you can just

> reload the YAML configuration server control page (you will need advanced mode).

Reviewer: ```suggestion
reload the YAML in the configuration server control page (you will need advanced mode).
```

> As of this release, besides the integration that already could be reloaded,

Reviewer: ```suggestion
As of this release, besides the integrations that already could be reloaded,
```

> the following integration can now reload their YAML configuration without
> restart Home Assistant:

Reviewer: ```suggestion
the following integration can now reload their YAML configuration without
a restart of Home Assistant:
```

- [MQTT][mqtt docs]
- [Ping][ping docs]
- [RESTful][rest docs]
- [Raspberry Pi GPIO][rpi_gpio docs]
- [SMTP][smtp docs]
- [Statistics][statistics docs]
- [Telegram][telegram docs]
- [Templates][template docs]
- [Trend][trend docs]
- [Universal Media Player][universal docs]

You can also reload an integration that is setup with the UI! This can be useful

> when it lost it's connection or is in an otherwise failed state. You can find

Reviewer: ```suggestion
when it lost its connection or is in an otherwise failed state. You can find
```

> the reload the button in the overflow menu on the integration card.

Reviewer: ```suggestion
the reload button in the overflow menu on the integration card.
```

<p class='img'>
<img src='/images/blog/2020-09-0.115/integrations-reload.png' alt='Screenshot of reloading an UI configured integration.'>
UI configured integration can now be reloaded as well!
</p>

## User password change

Another What the heck, that sounded so obvious: Being able to change a user's
password as the owner of the system. Right?!

You can now change every user's password from the UI when you are the owner of
the system!

<p class='img'>
<img src='/images/blog/2020-09-0.115/change-password.png' alt='Screenshot of change a password of a user account.'>
As a owner, you can now change a password of a user.
</p>

## Improved ways of exposing entities via Home Assistant Cloud

> This release brings an update to the way you can expose entities toGoogle

Reviewer: ```suggestion
This release brings an update to the way you can expose entities to Google
```

Assistant and Amazon Alexa via Home Assistant Cloud.

With the new panel, you can now set on a domain level if entities should be
exposed/not exposed by default. You can still override this on a per-entity
level for fine-grained control.

The default expose rules have also been updated to expose entities that work
best with voice assistants.

<img class='no-shadow' src='/images/blog/2020-09-0.115/cloud-expose-entities.png' alt='Screenshot of expose options on an entity.'>

## Add card by entities

Are you a bit overwhelmed by all the different types of cards Lovelace has? You
can now just select the entities you want to use for a card, and have Lovelace
suggest a card for you.

<p class='img'>
<img src='/images/blog/2020-09-0.115/add-card-by-entity-selection.png' alt='Screenshot of add card by entity selection view in Lovelace UI.'>
Select one or more entities to populate the card.
</p>

<p class='img'>
<img src='/images/blog/2020-09-0.115/add-card-by-entity-confirmation.png' alt='Screenshot of add card by entity confirmation view in Lovelace UI.'>

> A confirmation dialogue is shown before adding the suggested card.

Reviewer: ```suggestion
A confirmation dialog is shown before adding the suggested card.
```

for that specific item.

<details>
  <summary><b>Automations</b></summary>
  <p>

Previously an automation's `last_triggered` attribute was updated, and an
`automation_triggered` event was fired, whenever a trigger fired and the
conditions (if any) were true, regardless if the actions actually ran.

For example, in `single` mode, the actions won't run if they are still running
from a previous trigger event.

Now the attribute will be updated, and the event fired, only if the actions
actually run.

([@pnbruckner] - [#39323]) ([automation docs])

  </p>
</details>

<details>
  <summary><b>Axis</b></summary>
  <p>

Initial naming of events from VMD4 and Fence guard are now based on their
configured name on the device; `binary_sensor.m1065-lw_0_vmd4_camera1profile1`
is now `binary_sensor.m1065-lw_0_vmd4_profile_1` or `profile_1` can be whatever
the user chose to name the profile.

([@Kane610] - [#39699]) ([axis docs])

  </p>
</details>

<details>
  <summary><b>Broadlink</b></summary>
  <p>

**1. Devices are now configured via configuration flow**

To set up a Broadlink device, click _Configuration_ in the sidebar and click
_Integrations_.

> The devices will be imported from your configuration files to that page. If yousee

Reviewer: ```suggestion
The devices will be imported from your configuration files to that page. If you see
```

</details>

<details>
  <summary><b>Deutsche Wetter Dienst (DWD) Weather Warnings</b></summary>
  <p>

If you use entity state attributes of this integration in automations or scripts
need to adjust these to handle the changes.

- The `region_state` attribute has been removed, cause it is no longer available
  on the new API.
- All timestamps in the state attributes are now UTC and not local time anymore.

([@stephan192] - [#34820]) ([dwd_weather_warnings docs])

  </p>
</details>

<details>
  <summary><b>Emulated Hue</b></summary>
  <p>

By default, all lights and devices that do not support brightness adjustment are
exported as On/Off lights without a brightness property. When upgrading from
earlier versions of Home Assistant (0.112 and earlier), some devices may now be
reported by Alexa as non-responsive.

Alternatively, you can set the `lights_all_dimmable` configuration option to
continue reporting these devices as if they have a brightness setting.

**How to fix it once and for all**:

You need to have Alexa rediscover all devices and then remove the now
non-responding duplicates using the Alexa phone App. This can take quite a while
if you have lots of devices.

> An alternative would be to log to the Alexa web site and remove all the lights

Reviewer: ```suggestion
An alternative would be to log in to the Alexa web site and remove all the lights
```

instead and then re-discover them all.

To do so go to <https://alexa.amazon.com/spa/index.html#appliances>, or if not
logged in: <https://alexa.amazon.com> then select "Smart Home" -> "Devices" and
select "Remove All".

If you have multiple Echo devices on your network, it is possible that the
entries would continue to show as duplicates. This is due to an individual Echo
device caching the old list and re-using it.

The only known solution for this is to remove your echo devices from your Amazon
account, delete all the lights previously discovered by Alexa and then re-run
discovery.

This is a one-off requirement, unfortunately there's no other way to easily

> transitioned from the previously incorrect values reported by the Emulated Hue.

Reviewer: ```suggestion
transition from the previously incorrect values reported by the Emulated Hue.
```

is now hidden in the generated Lovelace view. If you want to continue to display
those you need to take control over your view with the 3 dots in the top right
corner of the Lovelace screen.

([@ludeeus] - [#6873]) ([lovelace docs])

  </p>
</details>

<details>
  <summary><b>MDI Icons</b></summary>
  <p>

The MDI icons are updated to version [5.5.55](https://dev.materialdesignicons.com/changelog#version-5.5.55),
this adds another 100 icons you can use!

In 5.5.55 there was 1 breaking change, if you used the icon `mdi:scooter` this
has been renamed to `mdi:human-scooter` and you need to adjust your
configuration.

All icons that were deprecated in [0.113.0](/blog/2020/07/22/release-113/#mdi-icons-updated)
have now been removed. Icons that were renamed or deleted in
version [5.0.45](https://dev.materialdesignicons.com/upgrade#4.9.95-to-5.0.45)
will no longer work.

  </p>
</details>

<details>
  <summary><b>Météo-France</b></summary>
  <p>

The attributes of `next_rain` has been reworked. In the previous version it was
a list of objects with changing keys (every 5 minutes) corresponding to a
UTC timestamp. This design was difficult to use in templates and automation.

> The new design will add a dedicated string attribute to have the the reference

Reviewer: ```suggestion
The new design will add a dedicated string attribute to have the reference
```

  <summary><b>Meteorologisk institutt (Met.no)</b></summary>
  <p>

While updating the integration, and its underlying libraries, to use the newer
API endpoint, some of the calculations and forecast aggregations were tweaked a
bit:

- Use hourly forecast for current weather, not daily.
- Ensure compared datetime objects are compared in the same time zone.
- Use highest resolution data from full 24 hours to calculate daily forecast
  min/max/sum values.

None of these changes are expected to break your setup, though the data
presented might look a little different due to the above.

In addition, all time stamps are now given in UTC. Automations that depend on
the `datetime` key under the state attribute `forecast` needs to be checked and
updated accordingly.

([@thimic] - [#39493]) ([met docs])

  </p>
</details>

<details>
  <summary><b>Netatmo</b></summary>
  <p>

The sensor for wind and gust angle is split up into two entities so that it now
returns the direction (e.g., `NE`) and the actual value (e.g., `178°`) rather
than a string containing both (e.g., `NE (123°)`).

([@cgtobi] - [#38627]) ([netatmo docs])

  </p>
</details>

<details>
  <summary><b>NZBGet</b></summary>
  <p>

> NZBGet is now available via the Integrations UI. This also means its no longer

Reviewer: ```suggestion
NZBGet is now available via the Integrations UI. This also means it's no longer
```

  </p>
</details>

<details>
  <summary><b>OAuth2 authentication and redirects</b></summary>
  <p>

Integrations using OAuth2 authentication now use the current request URL from
the browser as the redirect target, instead of the internal URL setting.

This matches the experience one would expect to happen and removes the need
to fiddle around with the internal URL setting.

However, this might require you to update application settings when
re-authenticating with existing services.

([@frenck] - [#38692])

  </p>
</details>

<details>
  <summary><b>Open Hardware Monitor</b></summary>
  <p>

In some locales numbers with decimals uses "," instead of "." and this causes
an issue when trying to use InfluxDB for example. This has been adjusted.

([@fillefilip8] - [#39030]) ([openhardwaremonitor docs])

  </p>
</details>

<details>
  <summary><b>OpenUV</b></summary>
  <p>

Support for configuring this integration has been fully removed.
If you have existing OpenUV configuration in your YAML configuration files,
you can safely remove that configuration.

([@bachya] - [#38857]) ([openuv docs])

  </p>
</details>

<details>
  <summary><b>OpenWeatherMap</b></summary>
  <p>

> The OpenWeatherMap integration can now be configured via the UI. After upgrading

Reviewer: ```suggestion
The OpenWeatherMap integration can now be configured via the UI. After upgrading,
```

  </p>
</details>

<details>
  <summary><b>Themes</b></summary>
  <p>

The code editor is now themeable, you can set the background color and the color
for the different code blocks. The default background color now is
`card-background-color`, for some themes this may conflict with the default code
colors. To get the old behavior back add `code-editor-background-color: white`
to your theme.

  </p>
</details>

<details>
  <summary><b>MDI Icons</b></summary>
  <p>

> The MDI icons are updated to version [5.5.55](https://dev.materialdesignicons.com/changelog#version-5.5.55), another 100 icons where added.

Reviewer 3: ```suggestion
The MDI icons are updated to version [5.5.55](https://dev.materialdesignicons.com/changelog#version-5.5.55), this adds another 100 icons you can use!

In 5.5.55 there was 1 breaking change, if you used the icon `mdi:scooter` this has been renamed to `mdi:human-scooter` and you need to adjust your configuration.

All icons that were deprecated in [0.113.0](https://www.home-assistant.io/blog/2020/07/22/release-113/#mdi-icons-updated) have now been removed.
```

Reviewer 2: Last line of suggestion: has -> have :)

URL or a list of them. If a single URL, `media_content_type` is `music`.
If a playlist, `media_content_type` is a `playlist`. If you are using
automations that use `media_content_id`, you should check if the
`media_content_type` is `music` or `playlist`.

([@rajlaud] - [#38214]) ([squeezebox docs])

  </p>
</details>

<details>
  <summary><b>Templates</b></summary>
  <p>

It is no longer necessary to provide a list of entities to monitor for each
template platform as automatic analysis can now find all entities that affect
the state in the template without manual setup.

This means you can now remove the `entity_id` option from your templates,
as it is now deprecated.

The template is now re-evaluated whenever an entity that can affect the
template state changes. New entities that can affect the template state are
automatically discovered each time the template is rendered. This change solves
a performance issue where the template would be re-rendered unnecessarily.

Please review the [`Working without entities`][working-without-entities] section
on the [`Binary Sensor Template`][working-without-entities] documentation for

> alternative ways to force templates entities to re-evaluate. This includes

Reviewer: ```suggestion
alternative ways to force template entities to re-evaluate. This includes
```

templates that rely on the use of `now()`.

If this change means you need to make adjustments, we have made it easier by
making template entities reloadable in the `YAML configuration reloading`
section under `Configure Home Assistant` -> `Server Controls`.

([@bdraco] - [#39382]) ([template docs])

[working-without-entities]: /integrations/binary_sensor.template/#working-without-entities

  </p>
</details>

<details>
  <summary><b>Themes</b></summary>
  <p>

The code editor is now themeable, you can set the background color and the color
for the different code blocks. The default background color now is

> `card-background-color`, for some themes this may conflict with the default code

Reviewer: ```suggestion
`card-background-color`. For some themes this may conflict with the default code
```
